boards/arm/sensortile_box_pro/sensortile_box_pro.dts

```diff
@@ -158,10 +158,28 @@ stm32_lp_tick_source: &lptim1 {
 };
 
 &spi1 {
-	pinctrl-0 = <&spi1_nss_pe12 &spi1_sck_pe13
-		     &spi1_miso_pe14 &spi1_mosi_pe15>;
+	pinctrl-0 = <&spi1_sck_pa5 &spi1_miso_pa6 &spi1_mosi_pa7>;
 	pinctrl-names = "default";
+	cs-gpios = <&gpioa 2 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
 	status = "okay";
+
+	bluenrg_lp_sensortile_pro: bluenrg_lp@0 {
+		compatible = "zephyr,bt-hci-spi", "st,hci-spi-v2";
+		reg = <0>;
+		irq-gpios = <&gpiod 1 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>;
+		reset-gpios = <&gpiod 4 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
```

Comment:
I don't think you want `GPIO_PULL_UP` here, the pin is set as output, what's the point of adding any bias? just `GPIO_ACTIVE_LOW`. There's other instances in the code base for stm32 boards with this as well, I think they should be fixed as well if you feel like following up on those too on a separate PR.

Doesn't really add anything in term of power consuption since the pin sits normally high but it just makes no sense.